<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NConfigWidget</class>
 <widget class="QWidget" name="NConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="appConfigFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="appStartScene">
        <property name="placeholderText">
         <string>scene name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="appSizeWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>scene</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="appSizeHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="sceneConfigTreeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>601</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>class</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>calss file</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>layout</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>page</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="pageConfigTreeWidget">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>class</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>class file</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>layout</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>appConfigFrame</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>NConfigWidget</receiver>
   <slot>contextMenuForConfigApp(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sceneConfigTreeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>NConfigWidget</receiver>
   <slot>contextMenuForConfigScene(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageConfigTreeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>NConfigWidget</receiver>
   <slot>contextMenuForConfigPage(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contextMenuForConfigApp(QPoint)</slot>
  <slot>contextMenuForConfigScene(QPoint)</slot>
  <slot>contextMenuForConfigPage(QPoint)</slot>
 </slots>
</ui>
